packages/manager/src/queries/placementGroups.ts

```diff
@@ -0,0 +1,135 @@
+import {
+  assignVmsToPlacementGroup,
+  createPlacementGroup,
+  deletePlacementGroup,
+  getPlacementGroup,
+  getPlacementGroups,
+  unassignVmsFromPlacementGroup,
+  updatePlacementGroup,
+} from '@linode/api-v4';
+import {
+  APIError,
+  Filter,
+  Params,
+  ResourcePage,
+} from '@linode/api-v4/lib/types';
+import { useMutation, useQuery, useQueryClient } from 'react-query';
+
+import { queryKey as PROFILE_QUERY_KEY } from './profile';
+
+import type {
+  CreatePlacementGroupPayload,
+  PlacementGroup,
+  UpdatePlacementGroupPayload,
+} from '@linode/api-v4';
+
+export const queryKey = 'placement-groups';
+
+export const usePlacementGroupsQuery = (
+  params: Params,
+  filter: Filter,
+  enabled: boolean = true
+) =>
+  useQuery<ResourcePage<PlacementGroup>, APIError[]>(
+    [queryKey, 'paginated', params, filter],
+    () => getPlacementGroups(params, filter),
+    { enabled, keepPreviousData: true }
+  );
```

Comment:
Completely optional, but if you have the time, it would be awesome if you could convert all of your new queries/mutations to use the "single object" notation. This will save us a lot of refactoring when we update React Query hopefully soon. 

In summary, instead of using many parameters, just pass a single object to `useQuery` with a `queryKey` and `queryFn`. Same with mutations, but mutations will use `mutationFn` I believe

```suggestion
  useQuery<ResourcePage<PlacementGroup>, APIError[]>({
    enabled,
    keepPreviousData: true,
    queryFn: () => getPlacementGroups(params, filter),
    queryKey: [queryKey, 'paginated', params, filter],
  });
```

Reply:
of course, will do that

Reply:
Yes, looks great! Thanks for doing that!